apps/web/src/pages/Pools/CreatePoolPage/CreatePoolPage.tsx

```diff
@@ -93,19 +93,19 @@ export const CreatePoolForm = ({
   };
 
   // recycled from EditPool
-  const classificationOptions: Option<string>[] = classificationsArray
-    .map(({ id, group, level, name }) => ({
+  const classificationOptions: Option[] = alphaSortOptions(
+    classificationsArray.map(({ id, group, level, name }) => ({
```

Comment:
I think this is redundant since the options are being sorted in the base `Select` component

Reply:
Oh yea 😆

Reply:
Fixed by [f44d52d](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9534/commits/f44d52d8513c01020f725bf092de88c08aa4067f)